src/test/scala/chiseltest/tests/ChiselEnumTest.scala

```diff
@@ -47,26 +46,40 @@ class ChiselEnumTest extends AnyFlatSpec with ChiselScalatestTester with Matcher
     }
   }
 
-  ignore should "peek enum literals" in {
+  it should "peek enum literals" in {
     test(new Module {
       val io = IO(new Bundle {
         val in = Input(UInt(2.W))
         val out = Output(EnumExample())
       })
-      io.out := io.in.asTypeOf(chiselTypeOf(io.out))
+      val (enum, _) = EnumExample.safe(io.in)
```

Comment:
enum is a reserved keyword I think? this may break in Scala 3

Reply:
I will change it.